app/models/metric/capture.rb

```diff
@@ -1,4 +1,5 @@
-module Metric::Capture
+class Metric
+class Capture
```

Comment:
This so you didn't have to indent the whole rest of the file?

Reply:
yea, I could do `class Metric::Capture`

Reply:
deleted